packages/@uppy/drop-target/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@uppy/drop-target",
+  "description": "Lets your users drag and drop and paste files on any of your existing DOM elements. Most likely document.body",
```

```suggestion
  "description": "Lets your users drag and drop files on a DOM element",
```
shorter = better(?)

might be good to update all the descriptions to a common format, "Uppy plugin for importing files from..." "Uppy plugin for receiving dropped files" etc. but that's a separate issue :)

---

Agreed! Thanks. Added to my todos.